activities/models/timeline_event.py

```diff
@@ -55,6 +55,7 @@ class Types(models.TextChoices):
 
     published = models.DateTimeField(default=timezone.now)
     seen = models.BooleanField(default=False)
+    dismissed = models.BooleanField(default=False)
```

Comment:
We might need to consider an index on this in future, but for now I think it's selective enough it should be fine.